tests/test_web/test_api_v1.py

```diff
@@ -904,8 +928,9 @@ def test_regenerate_bundle_missing_required_param(data, error_msg, db, auth_env,
     ),
 )
 @mock.patch('iib.web.api_v1.handle_regenerate_bundle_request')
+@mock.patch('iib.web.api_v1.messaging.send_message_for_state_change')
```

Comment:
Can we add an assertion to the non-success cases to ensure a message is not sent?